src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-utils.c

```diff
@@ -524,6 +524,10 @@ static void xclmgmt_reset_pci(struct xclmgmt_dev *lro)
 	xocl_pci_restore_config_all(lro);
 
 	xclmgmt_config_pci(lro);
+
+	if (xocl_pmc_enable_reset(lro) == -ENODEV) {
+		mgmt_info(lro, "no support for pmc pcie reset.\n");
```

Comment:
Most of the platforms do not have pmc. Could you move the print message inside pcm sub device?

Reply:
thanks, removed those logs.